app/brave_generated_resources.grd

```diff
@@ -468,6 +468,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         <message name="IDS_MUTE_TAB_MAC" desc="The Mac menu item to toogle active tab's mute state">
           Mute Tab
         </message>
+        <message name="IDS_COMMANDER_MENU_ITEM" desc="Text for the menu item to open the Commander">
```

Comment:
Instead of the new translation, can you use `IDS_IDC_COMMANDER`

Reply:
Nice!